src/features/environmentCreate/components/Specification/SpecificationCreate.tsx

```diff
@@ -103,16 +120,36 @@ export const SpecificationCreate = ({ onCreateEnvironment }: any) => {
     return stringify({ channels, dependencies, variables });
   };
 
+  const formatCodeLockfile = () => {
+    return stringify({
+      version: 1,
+      metadata: {},
+      package: []
+    });
+  };
```

Comment:
This is just to hint to the user which lockfile format is expected. Similar idea to `formatCode` above.